producers/src/main/resources/uk/gov/moj/sdt/producers/mbean/spring.context.xml

```diff
@@ -1,76 +1,76 @@
+		http://www.springframework.org/schema/aop/spring-aop.xsd">
+
+    <!-- Note all ids should be based on fully qualified names (interfaces
+        where this is not ambiguous) and all classes should have an interface. -->
+
+    <bean class="org.springframework.jmx.export.MBeanExporter">
+        <property name="registrationBehaviorName">
+            <value>REGISTRATION_REPLACE_EXISTING</value>
+        </property>
+        <property name="beans">
+            <map>
+                <entry key="bean:name=sdtProducersMetrics"
+                    value-ref="uk.gov.moj.sdt.utils.mbeans.api.ISdtMetricsMBean" />
+            </map>
+        </property>
+        <property name="assembler">
+            <bean
+                class="org.springframework.jmx.export.assembler.MethodNameBasedMBeanInfoAssembler">
+                <property name="managedMethods">
+                    <value>getTime,
+                        getOsStats,
+                        getBulkSubmitStats,
+                        getBulkFeedbackStats,
+                        getSubmitQueryStats,
+                        getStatusUpdateStats,
+                        getDomainObjectsStats,
+                        getDatabaseCallsStats,
+                        getDatabaseReadsStats,
+                        getDatabaseWritesStats,
+                        getActiveCustomersStats,
+                        getRequestQueueStats,
+                        getTargetAppStats,
+                        getErrorStats,
+                        getLastRefStats,
+                        getPerformanceLoggingString,
+                        uncache,
+                        setPerformanceLoggingFlags,
+                        reset,
+                        dumpMetrics
+                    </value>
+                </property>
+            </bean>
+        </property>
+    </bean>
+
+<!--    <bean class="org.springframework.jmx.export.MBeanExporter">-->
+<!--        <property name="registrationBehaviorName">-->
+<!--            <value>REGISTRATION_REPLACE_EXISTING</value>-->
+<!--        </property>-->
+<!--        <property name="beans">-->
+<!--            <map>-->
+<!--                <entry key="bean:name=sdtProducersManagement"-->
+<!--                    value-ref="uk.gov.moj.sdt.utils.mbeans.api.ISdtManagementMBean" />-->
+<!--            </map>-->
+<!--        </property>-->
+<!--        <property name="assembler">-->
+<!--            <bean-->
+<!--                class="org.springframework.jmx.export.assembler.MethodNameBasedMBeanInfoAssembler">-->
+<!--                <property name="managedMethods">-->
+<!--                    <value>uncache, -->
+<!--                        setMdbPoolSize,-->
+<!--                        requeueOldIndividualRequests,-->
+<!--                        processDLQRequest-->
+<!--                    </value>-->
+<!--                </property>-->
+<!--            </bean>-->
+<!--        </property>-->
+<!--    </bean>-->
```

Comment:
Any particular reason this section has been commented out?

Reply:
need to discuss about this